src/storage/secondary/transaction.rs

```diff
@@ -97,17 +97,11 @@ impl SecondaryTransaction {
         } else {
             return Ok(());
         };
-        let rowset_id = self.table.generate_rowset_id();
+        let rowset_id = self.table.get_current_rowset_id();
```

Comment:
This would cause problems in multi-thread cases. Better to store `rowset_id` in the memtable struct.

Reply:
updated